liquibase/dbchangelog.xml

```diff
@@ -0,0 +1,113 @@
+            <column name="fileurl_id" type="int">
+                 <constraints nullable="false"/>
+            </column>
+            <column name="childfileurl_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addForeignKeyConstraint baseTableName="fileurl_childs" baseColumnNames="fileurl_id" constraintName="fk_parent" referencedTableName="fileurls"
+                                 referencedColumnNames="id"></addForeignKeyConstraint>
+
+        <addForeignKeyConstraint baseTableName="fileurl_childs" baseColumnNames="childfileurl_id" constraintName="fk_child" referencedTableName="fileurls"
+                                 referencedColumnNames="id"></addForeignKeyConstraint>
+
+        <!-- Properties table -->
+        <createTable tableName="property">
+            <column name="property" type="varchar(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="value" type="varchar(1000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="updatedate" type="bigint"/>
+        </createTable>
+
+        <!-- Executions table -->
+        <createTable tableName="execution">
+            <column name="execution" type="varchar(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="result" type="clob" />
+            <column name="success" type="char(1)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="updatedate" type="bigint"/>
+        </createTable>
+
+        <!-- LogEvents table -->
+        <createTable tableName="logevents">
+            <column name="id" type="int">
+                <constraints primaryKey="true" primaryKeyName="pk_logevents"/>
+            </column>
+            <column name="org" type="varchar(200)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="repo" type="varchar(200)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="files" type="varchar(1000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="message" type="clob">
+                <constraints nullable="false"/>
+            </column>
+            <column name="commitdate" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="commits" type="varchar(500)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="varchar(500)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="rawdata" type="clob">
+                 <constraints nullable="false"/>
+            </column>
+            <column name="rendereddinghyfile" type="clob">
+            </column>
+        </createTable>
```

Comment:
What's the implication of adding an author field even if we don't use it yet? Will it be difficult to add later with liquidbase?

Reply:
I will add it now, in future we should add a changeset element with a new id.